snuba/migrations/migration.py

```diff
@@ -113,6 +113,9 @@ class ClickhouseNodeMigration(Migration, ABC):
     completely unrelated, they are probably better as separate migrations.
     """
 
+    forwards_local_first: bool = True
+    backwards_local_first: bool = True
```

Comment:
Why do we need to define two variables? Shouldn't backwards always the reverse of forwards? At least I can't think of a  valid scenario where that wouldn't be the case.

Reply:
It should be that way, but in our code there is nothing that enforces this. Backwards is another sequence or arbitrary SQL ops.

Reply:
I think the default should be that `backwards_local_first` is `False` if we have the default be that `forwards_local_first` is `True`.

Reply:
agreed. changed the defaults.